build.gradle

```diff
@@ -190,6 +190,10 @@ subprojects {
   task testlibJar(type: Jar, dependsOn: test) {
     classifier = 'testlib'
     from sourceSets.test.output
+
+    include('com/google/api/gax/rpc/testing/**')
```

Comment:
How exactly to define the external-facing gax testing utils here? I see this puts only the "testing" subfolders, but is that it? There are other infrastructure classes, which are not in testing subfodlers. For example `com.google.api.gax.core.FakeApiClock` or `com.google.api.gax.rpc.AccumulatingStreamObserver`.

Also, does this solve the other part of the issue (missing source files in the artifact)?

Reply:
@elharo to confirm whether the external-facing utils are limited to the `testing` packages.

The testlib sources are already in the sources jar, but I've added something to strip out unit tests as well.